x/ocap2/addons/recorder/fnc_init.sqf

```diff
@@ -150,7 +150,7 @@ if (GVAR(missionName) == "") then {
 [{
   if (
     QEGVAR(settings,saveOnEmpty) &&
-    !isNil QGVAR(startTime) && (GVAR(frameCaptureDelay) * GVAR(captureFrameNo)) / 60 >= GVAR(minMissionTime) && count allPlayers == 0
+    !isNil QGVAR(startTime) && (GVAR(frameCaptureDelay) * GVAR(captureFrameNo)) / 60 >= GVAR(minMissionTime) && count (call BIS_fnc_listPlayers) == 0
```

Comment:
```suggestion
    !isNil QGVAR(startTime) && (GVAR(frameCaptureDelay) * GVAR(captureFrameNo)) / 60 >= GVAR(minMissionTime) && count (call CBA_fnc_players) == 0
```
Only difference is CBA's doesn't report headless clients (which is probably what you want anyways), and it's faster I think.

Reply:
According to https://community.bistudio.com/wiki/allPlayers, BIS_fnc_listPlayers will exclude headless clients. But we're using CBA and I'd rather be safe. Good call